src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -4,8 +4,9 @@
     <EnableBinPlacing>false</EnableBinPlacing>
     <!-- This project should not build against the live built .NETCoreApp targeting pack as it contributes to the build itself. -->
     <UseLocalTargetingRuntimePack>false</UseLocalTargetingRuntimePack>
+    <!-- Use targeting pack references instead of granular ones in the project file. -->
+    <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
```

Comment:
This doesn't feel related.

Reply:
Unless the answer is that the deleted line was causing this property to be computed to false, and that's the whole reason that the deleted line was present?

Reply:
Right, before `IsSourceProject` was set to false which caused the `DisableImplicitAssemblyReferences` property to not be true. Now it is true so we need to set it to false same as the Microsoft.Extensions.* projects which disable this as well.